docs/install/index.md

```diff
@@ -65,31 +91,33 @@ pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.
 
 For previous versions, you can download here: [1.18.1](https://pypi.org/project/onnxruntime-gpu/1.18.1/), [1.18.0](https://pypi.org/project/onnxruntime-gpu/1.18.0/)
 
-#### Install ONNX Runtime GPU (ROCm)
-For ROCm, please follow instructions to install it at the [AMD ROCm install docs](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/). The ROCm execution provider for ONNX Runtime is built and tested with ROCm 6.0.0. 
-
-To build from source on Linux, follow the instructions [here](https://onnxruntime.ai/docs/build/eps.html#amd-rocm).
 
-### Install ONNX to export the model
+### Install ONNX Runtime QNN
 
 ```bash
-## ONNX is built into PyTorch
-pip install torch
+pip install onnxruntime-qnn
 ```
 
-```bash
-## tensorflow
-pip install tf2onnx
-```
+#### Install nightly
 
 ```bash
-## sklearn
-pip install skl2onnx
+pip install --pre --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ onnxruntime-qnn
 ```
 
+### Install ONNX Runtime GPU (ROCm)
+
+For ROCm, please follow instructions to install it at the [AMD ROCm install docs](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/). The ROCm execution provider for ONNX Runtime is built and tested with ROCm 6.0.0. 
```

Comment:
Our nuget for 1.20 uses 6.2.0.  I suggest to update the link and version number to 6.2.0 here:
https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.2.0/

Reply:
Fixed